airflow-core/src/airflow/ui/src/layouts/Nav/Nav.tsx

```diff
@@ -148,7 +148,14 @@ export const Nav = () => {
       <Flex alignItems="center" flexDir="column" width="100%">
         <Box mb={3}>
           <NavLink to="/">
-            <AirflowPin height="35px" width="35px" />
+            <AirflowPin
+              height="35px"
+              width="35px"
+              _hover={{
+                transform: "rotate(360deg)",
+                transition: "transform 0.8s ease-in-out"
+              }}
```

Comment:
Hey @choo121600, I helped on this in the Airflow 2 version of this fun enhancement. One consideration that we made there, that we should carry forward is respecting [the `prefers-reduced-motion` media query](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) that allows users to suppress non-essential animations. I _think_ this could be done with something like this, but I did not test it.
```suggestion
              sx={{
                '@media (prefers-reduced-motion: no-preference)': {
                  _hover: {
                    transform: "rotate(360deg)",
                    transition: "transform 0.8s ease-in-out"
                  }
                }
              }}
```

Reply:
Thanks for the review! In Chakra UI 3, the `sx={{ '@media (prefers-reduced-motion: no-preference)': … }}` is not supported, so I updated it using `_motionSafe` instead.

Reply:
https://github.com/user-attachments/assets/6e4222ca-d077-4d6f-b68f-ae63bb864297

Reply:
Oh, nice!